src/System.Management.Automation/engine/hostifaces/PSCommand.cs

```diff
@@ -359,6 +359,26 @@ public PSCommand AddParameter(string parameterName)
             return this;
         }
 
+        /// <summary>
+        /// Adds a <see cref="CommandParameter"/> instance to the last added command.
+        /// </summary>
+        internal PSCommand AddParameter(CommandParameter parameter)
+        {
+            if (_currentCommand == null)
```

Comment:
I notice that the other method override uses:
```
if (_currentCommand is null) ...
```
I am not sure of the semantic difference, but shouldn't we be consistent?

Reply:
That was a style change introduced by `https://github.com/PowerShell/PowerShell/pull/13277`, which changed all `== null` and `!= null` instances to `is null` and `is not null`. That PR was reverted by https://github.com/PowerShell/PowerShell/pull/13322 after a discussion within the maintainers because we don't think either style is universally preferred to the other, and there is not obvious reason to force the `is/is not` style.